src/client/eppo-client.ts

```diff
@@ -351,6 +358,90 @@ export default class EppoClient implements IEppoClient {
     );
   }
 
+  public getBanditAction(
+    flagKey: string,
+    subjectKey: string,
+    subjectAttributes: Attributes,
+    actions: Record<string, Attributes>, // TODO: ability to provide a set of actions with no context, or context broken out by numeric/categorical
```

Comment:
Future PR

Note in addition to being _more_ specific, in the name of developer experience I think we should also make it easy to be _less_ specific--e.g., just give an array of action names (which we'd dedupe) if there are no action contexts at play.

Reply:
Fair, but then there's nothing contextual about the bandit. Nothing breaks, but this is very much not the intended use case.

Reply:
It would still have subject context!